src/usb_moded-dbus.c

```diff
@@ -41,6 +42,25 @@
 static DBusConnection *dbus_connection_sys = NULL;
 extern gboolean rescue_mode;
 
+/**
+ * Issues "sig_usb_config_ind" signal.
+*/
+static void usb_moded_send_config_signal(const char *section, const char *key, const char *value)
+{
+  if (dbus_connection_sys)
+  {
+	DBusMessage* msg = dbus_message_new_signal(USB_MODE_OBJECT, USB_MODE_INTERFACE, USB_MODE_CONFIG_SIGNAL_NAME);
+	if (msg) {
+		dbus_message_append_args(msg, DBUS_TYPE_STRING, &section,
+		                              DBUS_TYPE_STRING, &key,
+		                              DBUS_TYPE_STRING, &value,
```

Comment:
All other calls that return config values (`mode_request`, `get_net_config`, `get_config`) return it as a string. I was following the pattern. I don't see any plausible scenario that would require `DBUS_TYPE_VARIANT` here.